tests/users/test_dao_filter_by_skill.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+
+from flask import json
+from tests.base_test_case import BaseTestCase
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from tests.test_data import user1, user2, user3
+from tests.test_utils import get_test_request_header
+
+
+class FilterUsersBySkill(BaseTestCase):
```

Comment:
Please follow the conventions used in other files
```suggestion
class TestFilterUsersBySkill(BaseTestCase):
```